DESCRIPTION

```diff
@@ -25,6 +25,7 @@ Imports:
     lubridate (>= 1.8.0),
     magrittr (>= 2.0.1),
     odbc (>= 1.3.1),
+    openxlsx,
```

Comment:
```suggestion
    openxlsx (>= 4.2.5),
```
Pick a reasonable version, I just picked the current latest. See the [openxlsx changelog](https://ycphs.github.io/openxlsx/news/index.html)